packages/theme/src/modules/roadmaps.ts

```diff
@@ -11,22 +11,58 @@ export interface Roadmap {
   name: string;
   url: string;
   color: string;
+  display: string;
+  index: number;
+}
+
+// New interface for pagination parameters
+export interface GetRoadmapsParams {
+  first?: number;
+  after?: string;
+}
+
+// New interface for the paginated response
+export interface PaginatedRoadmapsResponse {
+  data: Roadmap[];
+  page_info: {
+    count: number;
+    current_page: number;
+    has_next_page: boolean;
+  };
+  total_pages?: number | null;
+  total_count?: number | null;
 }
 
 /**
- *	Get all roadmaps
+ * Get all roadmaps with cursor-based pagination
  *
- * @returns {object} response
+ * @param {GetRoadmapsParams} params - Pagination parameters
+ * @returns {Promise<PaginatedRoadmapsResponse>} response
  */
-export const getAllRoadmaps = async () => {
-  return await axios({
+export const getAllRoadmaps = async (params: GetRoadmapsParams = {}): Promise<PaginatedRoadmapsResponse> => {
```

Comment:
You need to wrap `PaginatedRoadmapsResponse` type with `AxiosResponse` generic type.